Source/WebCore/dom/DocumentFullscreen.h

```diff
@@ -65,7 +65,7 @@ class DocumentFullscreen final : public CanMakeWeakPtr<DocumentFullscreen> {
     const Document& document() const { return m_document.get(); }
     Ref<Document> protectedDocument() const { return m_document.get(); }
     Page* page() const { return document().page(); }
-    LocalFrame* frame() const { return document().frame(); }
+    LocalFrame* frame() const;
```

Comment:
Why the change here?

Reply:
It failed to compile due to the `frame()` function being implemented in an inline header. Not sure why it came up now though to be honest